setup.py

```diff
@@ -4,60 +4,36 @@
 import os
 import sys
 from setuptools import setup, find_packages
-from typing import Tuple
 
 
-def package_metadata() -> Tuple[str, str]:
-    sys.path.insert(0, os.path.join("graspy", "version"))  # TODO: #454 Change path in https://github.com/microsoft/graspologic/issues/454
-    from version import name, version
-    sys.path.pop(0)
+MINIMUM_PYTHON_VERSION = 3, 6  # Minimum of Python 3.6
 
-    version_path = os.path.join("graspy", "version", "version.txt")
-    with open(version_path, "w") as version_file:
-        _b = version_file.write(f"{version}")
-    return name, version
+if sys.version_info < MINIMUM_PYTHON_VERSION:
+    sys.exit("Python {}.{}+ is required.".format(*MINIMUM_PYTHON_VERSION))
 
+sys.path.insert(0, os.path.join("graspy", "version"))
+# TODO: #454 Change path in https://github.com/microsoft/graspologic/issues/454
+from version import version
+sys.path.pop(0)
 
-PACKAGE_NAME, VERSION = package_metadata()
+version_path = os.path.join("graspy", "version", "version.txt")
+with open(version_path, "w") as version_file:
+    _b = version_file.write(f"{version}")
 
-DESCRIPTION = "A set of python modules for graph statistics"
 with open("README.md", "r") as f:
     LONG_DESCRIPTION = f.read()
-AUTHOR = ("Eric Bridgeford, Jaewon Chung, Benjamin Pedigo, Bijan Varjavand",)
-AUTHOR_EMAIL = "j1c@jhu.edu"
-URL = "https://github.com/neurodata/graspy"
-MINIMUM_PYTHON_VERSION = 3, 6  # Minimum of Python 3.6
-REQUIRED_PACKAGES = [
-    "networkx>=2.1",
-    "numpy>=1.8.1",
-    "scikit-learn>=0.19.1",
-    "scipy>=1.4.0",
-    "seaborn>=0.9.0",
-    "matplotlib>=3.0.0",
-    "hyppo>=0.1.3",
-]
-
-
-def check_python_version():
-    """Exit when the Python version is too low."""
-    if sys.version_info < MINIMUM_PYTHON_VERSION:
-        sys.exit("Python {}.{}+ is required.".format(*MINIMUM_PYTHON_VERSION))
-
-
-check_python_version()
 
 setup(
-    name=PACKAGE_NAME,
-    version=VERSION,
-    description=DESCRIPTION,
+    name="graspy",
+    version=version,
+    description="A set of python modules for graph statistics",
     long_description=LONG_DESCRIPTION,
     long_description_content_type="text/markdown",
-    author=AUTHOR,
-    author_email=AUTHOR_EMAIL,
+    author="Eric Bridgeford, Jaewon Chung, Benjamin Pedigo, Bijan Varjavand",
```

Comment:
what do we even want this to be going forward... there are many authors now...

Reply:
We can do something like "Graspologic maintainer team" and for email.. maybe a group email? not sure if its worth the trouble

Reply:
i guess we don't have to address this now in this PR but probably should at some point